pipeline/04-interpret.R

```diff
@@ -227,17 +227,30 @@ if (comp_enable) {
   # comp indexes by 1
   comps[[1]] <- comps[[1]] + 1
 
-  # Translate comp indexes to PINs
+  # Translate comp indexes to PINs and document numbers
   comps[[1]] <- comps[[1]] %>%
-    mutate(across(everything(), \(idx_row) {
-      training_data[idx_row, ]$meta_pin
-    })) %>%
+    mutate(
+      across(
+        starts_with("comp_idx_"),
+        \(idx_row) {
+          training_data[idx_row, ]$meta_pin
+        },
+        .names = "comp_pin_{str_remove(col, 'comp_idx_')}"
+      ),
+      across(
+        starts_with("comp_idx_"),
+        \(idx_row) {
+          training_data[idx_row, ]$meta_sale_document_num
+        },
+        .names = "comp_doc_no_{str_remove(col, 'comp_idx_')}"
```

Comment:
There's a variety of different ways we refer to this type of data -- alternately, `doc_no` or `instruno` or `document_num`. What's the best choice for this particular use-case?

Reply:
Let's stick with what's used in the model pipeline - `document_num`.

Reply:
Makes sense, done in d510fe79c7d13eb7810f5724b5b46b1615ad4f87.